lib/job-iteration/active_record_batch_enumerator.rb

```diff
@@ -40,7 +40,7 @@ def each
     end
 
     def size
-      @base_relation.count
+      (@base_relation.count + @batch_size - 1) / @batch_size # ceiling division
```

Comment:
Actually, leaning back towards using ` (@base_relation.count.to_f / @batch_size).ceil`, then we could probably drop the comment

Reply:
I prefer keeping the computation in integers rather than going to floats. 🤷